mne/io/eeglab/eeglab.py

```diff
@@ -390,6 +389,14 @@ def __init__(self, input_fname, montage, eog=(), event_id=None,
                 info, data, filenames=[input_fname], last_samps=last_samps,
                 orig_format='double', verbose=verbose)
 
+        # create event_ch from annotations
+        annot = read_annotations_eeglab(input_fname)
+        old_annotations = self.annotations
+        self.set_annotations(annot)
+        events, _ = events_from_annotations(self, event_id=event_id)
```

Comment:
not sure about the error you get - but this will not replicate the old behavior (see the issue I created) as eeglab event of type `'12'` will not get event value `12`, right?